exercises/APE5-6/APE05-06E1b.tex

```diff
@@ -0,0 +1,52 @@
+We can proceed exactly as for the case "a". We can first analyze the four pure strategies and we already find two Nash equilibria :
+
+- $\mathbf{\{x_1\}}\times\mathbf{\{x_2\}}$. Indeed step 3 implies :
+\begin{align*}
+    3 =w_1&\ge u_1(x_1,x_2)=2 \\
+    7=w_2&\ge u_2(x_1,y_2)=6 
+\end{align*}
+And so all the step are respected (note that step 1 and 2 are trivial for the pure strategies) 
+
+- $\mathbf{\{y_1\}}\times\mathbf{\{y_2\}}$. Indeed step 3 implies :
+\begin{align*}
+    7 =w_1&\ge u_1(x_1,y_2)=6 \\
+    3=w_2&\ge u_2(y_1,x_2)=2
+\end{align*}
+Again all the step are respected.
+
+
+\begin{itemize}
+  \item[$\bullet$] Randomized vs pure strategy supports
+\end{itemize}
+
+One can develop Payoff equation (step 2) as in Exercise "a" and see that all support does not lead to a Nash equilibrium.
+
+\begin{itemize}
+  \item[$\bullet$] Fully randomized strategy 
+\end{itemize}
+- $\mathbf{\{x_1,y_1\}}\times\mathbf{\{x_2,y_2\}}$.
+
+Again we first check the payoff equations (step 2), for the player 1 ($i=1$) we have : 
+\begin{align*}
+ \sigma_2(x_2)u_1(x_1,x_2) + \sigma_2(y_2)u_1(x_1,y_2) &= \sigma_2(x_2)u_1(y_1,x_2) + \sigma_2(y_2)u_1(y_1,y_2) \\
+ 3\sigma_2(x_2)+6\sigma_2(y_2)&= 2\sigma_2(x_2) + 7\sigma_2(y_2) \\
+ \sigma_2(x_2)&=\sigma_2(y_2)
+\end{align*}
+
+Moreover we know that $\sigma_2(x_2) + \sigma_2(y_2) = 1$ and so we find $\sigma_2(x_2)=\frac{1}{2}$ and $\sigma_2(x_2)=\frac{1}{2}$
+
+Now for the player 2 ($i=2$) we have : 
+\begin{align*}
+ \sigma_1(x_1)u_2(x_1,x_2) + \sigma_1(y_1)u_2(y_1,x_2) &= \sigma_1(x_1)u_2(x_1,y_2) + \sigma_1(y_1)u_2(y_1,y_2) \\
+ 7\sigma_1(x_1)+2\sigma_1(y_1)&= 6\sigma_1(x_1) + 3\sigma_1(y_1) \\
+ \sigma_1(x_1)&=\sigma_1(y_1)
+\end{align*}
+
+We also know that $\sigma_1(x_1) + \sigma_1(y_1) = 1$ and so we find $\sigma_1(x_1)=\frac{1}{2}$ and $\sigma_1(y_1)=\frac{1}{2}$
+
+To conclude we have 3 Nash equilibrium :
+\begin{itemize}
+  \item[$\bullet$]  $\sigma_1 = \left(x_1,x_2\right)$ with $w_1= \left(3,7\right)$
```

Comment:
You cannot call it \sigma_1 since this means sigma for player 1. you can so \sigma^1 and w^1 though